cmd/monaco/generate/dependencygraph/command.go

```diff
@@ -71,6 +72,8 @@ func Command(fs afero.Fs) (cmd *cobra.Command) {
 
 	cmd.Flags().StringVarP(&outputFolder, "output-folder", "o", "", "The folder generated dependency graph DOT files should be written to. If not set, files will be created in the current directory.")
 
+	cmd.Flags().BoolVar(&jsonIDs, "json-ids", false, "Set to generate a DOT file using a JSON representation of coordinates as node IDs, instead of the string representation. This can be useful when processing generated DOT files automatically.")
+
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&jsonIDs, "json-ids", false, "Set to generate a DOT file using a nodes IDs encoded as JSON instead of the default string representation. This can be useful when processing generated DOT files automatically.")

```